sdk/scheduler/src/main/java/com/mesosphere/sdk/specification/DefaultServiceSpec.java

```diff
@@ -63,7 +68,8 @@ public DefaultServiceSpec(
             @JsonProperty("web-url") String webUrl,
             @JsonProperty("zookeeper") String zookeeperConnection,
             @JsonProperty("pod-specs") List<PodSpec> pods,
-            @JsonProperty("replacement-failure-policy") ReplacementFailurePolicy replacementFailurePolicy) {
+            @JsonProperty("replacement-failure-policy") ReplacementFailurePolicy replacementFailurePolicy,
+            @JsonProperty("gpu-optin") Boolean gpuFlag) {
```

Comment:
Again, this element should not be defined at the `Service` level, but at the `ResourceSet` level.